spec/lib/driver_response_spec.rb

```diff
@@ -16,7 +16,7 @@
     let(:driver) { client_response.first }
 
     it 'client response will not be empty' do
-      expect(client_response.empty?).to be_falsey
+      is_expected.not_to be_empty
```

Comment:
Please ignore the previous comment, I see that it is referenced in a nested describe.